test/ipynb/mpl/circuit/test_circuit_matplotlib_drawer.py

```diff
@@ -618,15 +620,18 @@ def test_user_style(self):
 
     def test_subfont_change(self):
         """Tests changing the subfont size"""
-        circuit = QuantumCircuit(3)
-        circuit.h(0)
-        circuit.x(0)
-        circuit.u(pi / 2, pi / 2, pi / 2, 1)
-        circuit.p(pi / 2, 2)
-        style = {"name": "iqx", "subfontsize": 11}
-
-        self.circuit_drawer(circuit, style=style, filename="subfont.png")
-        self.assertEqual(style, {"name": "iqx", "subfontsize": 11})  # check does not change style
+        for name in ["iqx", "iqx-dark"]:
+            with self.subTest(name=name):
+                circuit = QuantumCircuit(3)
+                circuit.h(0)
+                circuit.x(0)
+                circuit.u(pi / 2, pi / 2, pi / 2, 1)
+                circuit.p(pi / 2, 2)
+                style = {"name": name, "subfontsize": 11}
+
+                self.circuit_drawer(circuit, style=style, filename="subfont.png")
```

Comment:
This causes a naming clash as you're generating 'subfont.png' twice which causes the snapshot test to fail. I don't think it's strictly necessary to test the subfontsize with the new dark theme anyway but i guess it doesn't hurt

```suggestion
                self.circuit_drawer(circuit, style=style, filename="f"{name}_subfont.png"")
```